lib/installer.js

```diff
@@ -391,7 +391,7 @@ class Installer {
     this.log('verbose', 'tinkifyBins', 'convering installed bins to use tink:', bins)
     return BB.map(bins, async bin => {
       const real = await realpathAsync(bin)
-      const data = (await readFileAsync(real, 'utf8')).replace(/(#!\s*.*\s*)node/g, '$1tish')
+      const data = (await readFileAsync(real, 'utf8')).replace(/^(#!.*\b)node([^-\w]|$)/g, '$1tish$2')
       await writeFileAsync(real, data, 'utf8')
     }, { concurrency: 50, Promise: BB })
```

Comment:
What's tish? 🤩

Reply:
It's a script that shorthands `tink sh`, since you can't have arguments in some key places (like shebangs in some platforms). Also just a useful shorthand for it.

Reply:
I :heart: that mashup!